### PR TITLE
[KYUUBI #1058] Replace feature_request.md with feature-request.yml

### LINKED ISSUE #1058: Replace feature_request.md with feature-request.yml
Follow what we have done in https://github.com/apache/incubator-kyuubi/pull/1036 and migrate the feature template to yaml format

the text areas can follow the sections below

### PR DESCRIPTION
**Pull request by byyue**
### _Why are the changes needed?_

Close #1058.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Review by yaooqinn — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Kyuubi Feature Request
+title: "[feature] "
```
Comment (yaooqinn) [inline .github/ISSUE_TEMPLATE/feature-request.yml:19]: [FEATURE]

Comment (byyue) [reply .github/ISSUE_TEMPLATE/feature-request.yml]: Updated.

**Comment by yaooqinn** [comment]
thanks @byyue would you like to send a PR for the umbrella.md?

**Comment by byyue** [comment]
> thanks @byyue would you like to send a PR for the umbrella.md?

Sure. I deem the current umbrella.md quite brief. Do you want to add some extra content or just keep it as it is?

**Comment by yaooqinn** [comment]
> Sure. I deem the current umbrella.md quite brief. Do you want to add some extra content or just keep it as it is?

Yea, what do you suggest to add?

**Comment by byyue** [comment]
> > Sure. I deem the current umbrella.md quite brief. Do you want to add some extra content or just keep it as it is?
> 
> Yea, what do you suggest to add?

I just fine-tuned the format. Please see #1077.
